app/editor/src/components/form/error/Error.tsx

```diff
@@ -0,0 +1,15 @@
+import { Show } from 'tno-core';
+
+export interface IErrorProps {
+  /** The error message that needs to be displayed */
+  error?: string;
+}
+
+/** Simple component to conditionally display an error message */
+export const Error: React.FC<IErrorProps> = ({ error }) => {
```

Comment:
You could extend this component and make one for Formik.  It would just need the `name` property to extract the appropriate error.

Reply:
Good call